src/fund/fees/PerformanceFee.sol

```diff
@@ -33,7 +33,7 @@ contract PerformanceFee is DSMath {
 
     /// @notice Assumes management fee is zero
     function feeAmount() external returns (uint feeInShares) {
-        Shares shares = Shares(IHub(Spoke(msg.sender).getHub()).shares());
+        Shares shares = Shares(IHub(Spoke(msg.sender).HUB()).shares());
```

Comment:
Because of how we're using interfaces (as representations of the surface area of interactions between core contracts), we don't want to add new functions to an interface based on the needs of plugin contracts. It would be fine in this case (for this function), but in principle we're using full contract imports in plugins so that we don't accidentally for a function into the interface that doesn't need to be there.